elastic/indexer/model.go

```diff
@@ -0,0 +1,6 @@
+package indexer
+
+type ElasticIndexer struct {
```

Comment:
I don't think `ElasticIndexer` describes the purpose of this struct well; what do you think about something like `BulkIndexDoc`?

Reply:
is better, I couldn't think of anything better :)